docs/guide/adapters/README.md

```diff
@@ -145,6 +145,30 @@ _(in order of first appearance)_
     ![](../../images/jetstick_z2.png)
     </details>
 
+  * <details>
+    <summary>SMLIGHT SLZB-07p7 CC2652P7 Zigbee USB adapter - the first CC2652P7 USB adapter</summary>  
```

Comment:
```suggestion
    <summary>SMLIGHT SLZB-07p7 CC2652P7 Zigbee USB adapter</summary>  
```

Reply:
@gam-st also check this comment

Reply:
> @gam-st also check this comment

done https://github.com/Koenkk/zigbee2mqtt.io/pull/2718/commits/837a91965a57f4151b635eea566e6a2b8a987c3b